advocacy_docs/partner_docs/ThalesCipherTrustManager/04-ConfiguringThalesCipherTrustManager.mdx

```diff
@@ -0,0 +1,91 @@
+- Thales CipherTrust Manager 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server version 15.2 instance.
+- Thales CipherTrust Manager installed and deployed per your VM environment.
+
+## Configure Thales CipherTrust Manager for EDB Postgres Advanced Server
+
+After you have successfully installed Thales CipherTrust Manager you will then need to follow a few steps so you can create certificates for use.
+
+## Login to Thales CipherTrust Manager UI and Create User
+
+1. Using your static IP address you defined in the Thales CipherTrust Manager setup and configuration, log in to the Thales CipherTrust Manager UI via https and your previously configured Username and Password.
+    
+    ![Thales CipherTrust Manager Login](Images/ThalesCipherTrustLoginScreen.png)
+
+2. You will then navigate to the Access Management section and select `Users`.
+
+3. From here you will select `Add Users` and provide all of the required information:
+- Username
+- Password
+
+4. Click on `Add User`.
+
+    ![Thales CipherTrust Manager Add User](Images/ThalesCipherTrustAddUser.png)
+
+5. Click on the `...` beside the user you just created and selected the View/Edit option.
+
+6. Click on `Expand All` and select/add the following groups for the user:
+- `Key Admins`
+- `Key Users`
+- `All Clients`
+
+7. The user that you created now has the appropriate settings and permissions to view/download keys.
+
+## Create and Download Keys on Thales CipherTrust Manager UI
+
+### Download the CA Key
+
+This is the first in a series of keys that you will need for your KMIP server and pykmip.conf file.
+
+1. Navigate to CA on the left-hand side and click on `Local` to be taken to the Local Certificate Authorities page.
+
+2. Click on the `...` on the far right side and click `Download`.
+
+    ![Thales CipherTrust Manger CA Certificate](Images/ThalesCipherTrustCACert.png)
+
+3. For your download change the name from Certificate.pem to ca.pem
+
+### Create and Download the Key.pem and Cert.pem
+
+These two keys are the other keys you will need for your KMIP server and pykmip.conf file.
+
+1. Navigate back to the Thales CipherTrust Manager main page.
+
+2. Once you are back to the main page, select `KMIP` from options under Products.
+
+3. Click on `Client Profile` and select `Add Profile`.
+
+4. Give the profile a name and select `Save`.
+
+    ![Thales CipherTrust Manager Add Client Profile](Images/ThalesCipherTrustManagerAddClientProfile.png)
+
+5. Click on `Registration Token` and select `New Registration Token`.
+
+6. Click on Begin and then follow the below steps to configure your token:
+- Give the token a name
+- Select `local CA`
+- Select the Client Profile you created above
+- Select `Create Token`
+- Copy the Token that was created
+- Click on `Done`
+
+7. Navigate to Registered Clients and select `Add Client`.
+
+8. Paste the token that you created above into the Add Client window and then press `Save` and download the key.pem and cert.pem files.
```

Comment:
where did key.pem and cert.pem come from?  Are these default names or do I pick?

Reply:
These are the names that PYKMIP uses for the certs in the pykmip.conf file.  And yes I can make a step to note where the certs are saved.